sw/device/lib/dif/dif_hmac.h

```diff
@@ -0,0 +1,138 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 1.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef SW_DEVICE_LIB_DIF_HMAC_DEVICE_H_
+#define SW_DEVICE_LIB_DIF_HMAC_DEVICE_H_
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "sw/device/lib/base/mmio.h"
+
+/**
+ * Supported HMAC operations.
+ */
+typedef enum dif_hmac_ops {
+  DIF_HMAC_OP_HMAC = 0,
+  DIF_HMAC_OP_SHA256 = 1
```

Comment:
The format for the constants is `kDifHmacOpSha256`, consider naming it `kDifHmacOpsSha256` instead to match the type name.

I don't really like camel case in C, but all the rest of the OT uses this stye, so we should stick with it. And that is a preferred way described in the style guide.